scripts/deploy.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+const Client = require('ftp-deploy');
+const client = new Client();
+
+const config = {
+  user: "arteverest_edbadge",
```

Comment:
@vergilius we want to commit it? is it safe?

Reply:
yeah, if no password is sent it should be okay

Reply:
I can extract it tomorrow, so nothing is in the repo